exporter/signalfxexporter/config.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package signalfxexporter
+
+import (
+	"time"
+
+	"github.com/open-telemetry/opentelemetry-collector/config/configmodels"
+)
+
+// Config defines configuration for SignalFx exporter.
+type Config struct {
+	configmodels.ExporterSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct.
+
+	// AuthToken is the authentication token provided by SignalFx.
+	AuthToken string `mapstructure:"auth_token"`
+
+	// Realm is the SignalFx realm where data is going to be sent to. The
+	// default value is "us0"
+	Realm string `mapstructure:"realm"`
```

Comment:
There should also be an option for specifying the full path to the ingest URL so that the user isn't restricted to a fixed URL template (e.g. for sending to our lab environment, which doesn't follow the template)